### DISCUSSION TITLE
Nested tags -> folders?

### DISCUSSION THREAD

**Discussion by DamianMehers** [akosbalasko/yarle#267] [Q&A] [answered]
Hi,

As a heavy nested tag user, I'm looking for a tool that exports my tag hierarchy as nested folders, duplicating notes/files where they have multiple tags.

Does this do anything like that?

Thanks,
     Damian

**Comment by akosbalasko** [comment, marked as answer]
Hi @DamianMehers ,

nope, unfortunatelly it's not supported.
Take a look around in the market, and if you don't find anything with this feature, you can submit it as a feature request in the issues page. 

Cheers,
akos
